src/entity/user.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  Connection,
+  BaseEntity,
+} from "typeorm";
+
+@Entity()
+export class User extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  firstName: string;
+
+  @Column()
+  lastName: string;
+
+  @Column()
+  isActive: boolean;
+}
+
+export async function createUser(database: Connection) {
```

Comment:
é que agora você está usando o createUser logo dentro do .then do createConnection, mas quando for usar de outros lugares pode ser um tanto quanto trabalhoso ficar passando a connection (o objeto `database`). 
Felizmente o typeorm oferece o método `getConnection()` para que não seja necessário fazer isso o/
https://github.com/typeorm/typeorm/blob/master/docs/connection.md